plugins/janus_videoroom.c

```diff
@@ -2292,6 +2269,36 @@ void janus_videoroom_incoming_rtp(janus_plugin_session *handle, int video, char
 	if(!session || session->destroyed || !session->participant || session->participant_type != janus_videoroom_p_type_publisher)
 		return;
 	janus_videoroom_participant *participant = (janus_videoroom_participant *)session->participant;
+	if(participant->audio_active) {
+		int level = 0;
+		if(janus_rtp_header_extension_parse_audio_level(buf, len, participant->audio_level_extmap_id, &level) == 0) {
+			// JANUS_LOG(LOG_INFO, "Audio level is %d\n", level);
+			participant->audio_dBov_sum = participant->audio_dBov_sum + level;
+			participant->audio_active_packets = participant->audio_active_packets + 1;
+		// 2 seconds of talking (100 packets) with average of ~25 dBow
+			if(participant->audio_active_packets == 100) {
+				if(participant->audio_dBov_sum < 2500) {
```

Comment:
The two settings above should be made configurable as well, so that people can fine tune the behaviour themselves depending on their application, their needs or their taste.